ts/components/ui/Markdown/handlers/internalLink.ts

```diff
@@ -82,6 +83,11 @@ const fciRoutesToNavigationLink: Record<string, string> = {
   [FCI_ROUTES.MAIN]: "/fci/main"
 };
 
+const idPayOnboardingRoutesToNavigationLink: Record<string, string> = {
```

Comment:
@CrisTofani do we still need to also add internal links in this file?

Reply:
I thought it was nice to have a backwards compatibility considering the low cost (few lines of code). If this is not necessary I'll remove it from the PR.

Reply:
Agree, there's no need to add the link here since no previous version of the app supported this route 😄 

we can safely remove the input on this legacy file